src/main/java/org/broadinstitute/dsm/db/dao/fieldsettings/FieldSettingsDao.java

```diff
@@ -99,21 +101,20 @@ public int delete(int id) {
                 try(ResultSet rs = stmt.executeQuery()) {
                     while (rs.next()) {
                         fieldSettingsByOptions.add(
-                                new FieldSettingsDto(
-                                        rs.getInt(FIELD_SETTINGS_ID),
-                                        rs.getInt(DDP_INSTANCE_ID),
-                                        rs.getString(FIELD_TYPE),
-                                        rs.getString(COLUMN_NAME),
-                                        rs.getString(COLUMN_DISPLAY),
-                                        rs.getString(DISPLAY_TYPE),
-                                        rs.getString(POSSIBLE_VALUES),
-                                        rs.getString(ACTIONS),
-                                        rs.getBoolean(READONLY),
-                                        rs.getInt(ORDER_NUMBER),
-                                        rs.getBoolean(DELETED),
-                                        rs.getLong(LAST_CHANGED),
-                                        rs.getString(CHANGED_BY)
-                                )
+                                new FieldSettingsDto.Builder(rs.getInt(DDP_INSTANCE_ID))
+                                        .withFieldSettingsId(rs.getInt(FIELD_SETTINGS_ID))
+                                        .withFieldType(rs.getString(FIELD_TYPE))
+                                        .withColumnName(rs.getString(COLUMN_NAME))
+                                        .withColumnDisplay(rs.getString(COLUMN_DISPLAY))
+                                        .withDisplayType(rs.getString(DISPLAY_TYPE))
+                                        .withPossibleValues(rs.getString(POSSIBLE_VALUES))
+                                        .withActions(rs.getString(ACTIONS))
+                                        .withReadOnly(rs.getBoolean(READONLY))
+                                        .withOrderNumber(rs.getInt(ORDER_NUMBER))
+                                        .withDeleted(rs.getBoolean(DELETED))
+                                        .withLastChanged(rs.getLong(LAST_CHANGED))
+                                        .withChangedBy(rs.getString(CHANGED_BY))
+                                .build()
```

Comment:
seeing that this is reused, maybe we could put it into a method, so that we do not need to copy/paste?

Reply:
I think main purpose of Builder pattern is that not to have such method, just because several `DAO` method uses all the columns from `field_settings` table does not worth to create such method in my opinion, later we may have queries like which may only return only 3 column(`ddp_instance_id`, `field_settings_id`, `column_name` for example), that method will not work on such cases and may produce confusion, that's why Builder is more handy, what do you think?

Reply:
👍